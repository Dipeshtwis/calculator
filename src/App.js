import logo from './logo.svg';
import './App.css';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Calculator milestone 1 <code></code>.
        </p>
      </header>
    </div>
  );
}

export default App;
