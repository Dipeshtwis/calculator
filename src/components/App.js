function App() {
  return (
    <React.Fragment>
      <Display />
      <ButtonPanel />
    </React.Fragment>
  );
}

export default App;
