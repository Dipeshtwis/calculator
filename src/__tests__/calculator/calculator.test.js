/* eslint-disable no-unused-vars */
import React from 'react';
import renderer from 'react-test-renderer';
import App from '../../components/App';

it('renders correctly', () => {
  const app = renderer.create(<App />).toJSON();
  expect(app).toMatchSnapshot();
});