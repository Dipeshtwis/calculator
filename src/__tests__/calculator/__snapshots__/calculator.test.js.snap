// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div>
    <h1>
      Let's do some math!
    </h1>
  </div>,
  <div
    className="root-calc"
  >
    <h1
      className="input-value"
    >
      0
    </h1>
    <div
      className="calc"
    >
      <div
        className="row0"
      >
        <button
          className="operand"
          onClick={[Function]}
        >
          AC
        </button>
        <button
          className="number"
          onClick={[Function]}
        >
          +/-
        </button>
        <button
          className="number"
          onClick={[Function]}
        >
          %
        </button>
        <button
          className="operand"
          onClick={[Function]}
        >
          รท
        </button>
      </div>
      <div
        className="row1"
      >
        <button
          className="number"
          onClick={[Function]}
        >
          7
        </button>
        <button
          className="number"
          onClick={[Function]}
        >
          8
        </button>
        <button
          className="number"
          onClick={[Function]}
        >
          9
        </button>
        <button
          className="operand"
          onClick={[Function]}
        >
          x
        </button>
      </div>
      <div
        className="row2"
      >
        <button
          className="number"
          onClick={[Function]}
        >
          4
        </button>
        <button
          className="number"
          onClick={[Function]}
        >
          5
        </button>
        <button
          className="number"
          onClick={[Function]}
        >
          6
        </button>
        <button
          className="operand"
          onClick={[Function]}
        >
          -
        </button>
      </div>
      <div
        className="row3"
      >
        <button
          className="number"
          onClick={[Function]}
        >
          1
        </button>
        <button
          className="number"
          onClick={[Function]}
        >
          2
        </button>
        <button
          className="number"
          onClick={[Function]}
        >
          3
        </button>
        <button
          className="operand"
          onClick={[Function]}
        >
          +
        </button>
      </div>
      <div
        className="row4"
      >
        <button
          className="number"
          onClick={[Function]}
        >
          0
        </button>
        <button
          className="number"
          onClick={[Function]}
        >
          .
        </button>
        <button
          className="operand"
          onClick={[Function]}
        >
          =
        </button>
      </div>
    </div>
  </div>,
]
`;
